posts/2023-01-09_welcome/index.qmd

```diff
@@ -4,8 +4,8 @@ author:
   - name: Ben Straub
 
 description: "Short, fun and user-driven content around the pharmaverse."
-date: "2023-06-30"
-categories: [Blogging]
+date: "2023-06-28"
+categories: [R]
```

Comment:
I think this is not a tag we have on our taglist: 
c(
  "metadata", "submission", "qc", "ADaMs", "SDTMs", "community",
  "conferences", "admiral", "roak", "xportr", "metatools", "metacore"
)

Reply:
changed to community